src/controller/BridgeGame.js

```diff
@@ -0,0 +1,125 @@
+import SYSTEM from '../constants/System.js';
+import Bridge from '../model/Bridge.js';
+import Move from '../model/Move.js';
+import GameCommandValidator from '../utils/validators/GameCommand.js';
+import MovingValidator from '../utils/validators/Moving.js';
+import InputView from '../view/InputView.js';
+import OutputView from '../view/OutputView.js';
+
+/**
+ * 다리 건너기 게임을 관리하는 클래스
+ */
+class BridgeGame {
+  #inputView;
+
+  #outputView;
+
+  #model;
+
+  static SUCCESS = '성공';
+
+  static FAILURE = '실패';
+
+  constructor() {
+    this.#inputView = InputView;
+    this.#outputView = OutputView;
+    this.#model = new Move();
+  }
+
+  async startGame() {
+    this.#outputView.printStart();
+    const bridge = await this.getBridge();
+    const result = await this.#move(bridge);
+
+    this.#outputView.printResult(result, this.#model.getTryCount(), BridgeGame.SUCCESS);
+  }
+
+  async getBridge() {
+    const bridgeSize = await this.#handleError(() => this.#readBridgeSize());
+    const bridge = bridgeSize.makeBridge();
+
+    return bridge;
```

Comment:
```suggestion
    return bridgeSize.makeBridge();
```

p5 - 요거가 조금 더 깔끔할거같아요